src/client/pydaos/SConscript

```diff
@@ -12,7 +12,10 @@ def build_shim_module(env, lib_prefix, version):
 
     if version == "2.7":
         tgt_name = 'pydaos_shim_27'
-        new_env.ParseConfig("pkg-config --cflags --libs python2")
+
+        if not GetOption('clean'):
```

Comment:
Is there a reason you haven't added this to the python3 branch?

Reply:
By adding it to the python 3 branch, will prevent scons from cleaning the python 3 targets

Reply:
That doesn't explain why python2.7 and python3 and handled differently.

Are we saying that python3 is mandatory, and python 2.7 is optional?  If that's the case then a comment to that effect would be useful.

Reply:
Yes, do you think that a warning/info message at build time would be appropriate?
Something like this on the Python 2 branch:

Info: Python 2 not found. Skipping Python 2 targets

Reply:
No, this is legacy so I don't think we want to warn if it's not there but we should comment the code for the next person to read this.

Reply:
This placement around ParseConfig should have no effect on whether it gets cleaned or not so it could be wrapped around both python versions.  That said, if only 2.7 is optional, it is fine here.